src/modules/host_button/index.js

```diff
@@ -65,16 +65,19 @@ class HostButtonModule {
 
         const channelId = currentChannel.id;
 
-        const tmiSession = twitch.getCurrentTMISession();
-        if (!tmiSession) return;
-
-        tmiSession._tmiApi
-            .get('/hosts', {host: userId})
-            .then(data => {
-                if (!data.hosts || !data.hosts.length) return;
-                hosting = data.hosts[0].target_id === channelId;
-                this.updateHostButtonText();
-            });
+        let counter = 0;
+        const interval = setInterval(() => {
```

Comment:
hacky solution, I'd rather load this on chat load and check if it's the channel page